xbmc/settings/dialogs/GUIDialogSettingsBase.h

```diff
@@ -140,8 +140,8 @@ class CGUIDialogSettingsBase
    */
   virtual void OnClick(BaseSettingControlPtr pSettingControl);
 
-  void UpdateSettingControl(const std::string &settingId);
-  void UpdateSettingControl(BaseSettingControlPtr pSettingControl);
+  void UpdateSettingControl(const std::string &settingId, bool updateDisplayOnly = false);
```

Comment:
> Left-align `*` and `&` to the base type they modify.

https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md#52-pointer-and-reference-types

Reply:
Fixed